cdap-docs/reference-manual/source/characters.rst

```diff
@@ -12,10 +12,10 @@ Supported Characters
 
 The Cask Data Application Platform (CDAP) has naming conventions for different components of CDAP.
 
-Streams
--------
+Streams and Metadata
```

Comment:
What should we title it then? I'm at a bit of a loss, as it is the naming for just Streams and Metadata. And, namespaces is a more-restrictive sub-set. Should we list them by who uses them, or give the sets different names, and then mention who uses them?